fingerprint/infra/scanner/src/main/java/com/simprints/fingerprint/infra/scanner/v2/scanner/ScannerExtendedInfoReaderHelper.kt

```diff
@@ -152,25 +141,17 @@ class ScannerExtendedInfoReaderHelper(
     }
 
     private suspend fun getHardwareVersionInfo(): GetHardwareVersionResponse {
-        return sendRootModeCommandAndReceiveResponse<GetHardwareVersionResponse>(
+        return rootMessageChannel.sendRootModeCommandAndReceiveResponse<GetHardwareVersionResponse>(
             GetHardwareVersionCommand()
-        ).await()
+        )
     }
 
     suspend fun getExtendedVersionInfo(): GetExtendedVersionResponse {
-        return sendRootModeCommandAndReceiveResponse<GetExtendedVersionResponse>(
+        return rootMessageChannel.sendRootModeCommandAndReceiveResponse<GetExtendedVersionResponse>(
```

Comment:
Not that it's a big deal but there's a redundancy in having to specify the mode in both the channel and in the function.

Reply:
Good point, I updated the method to be `sendCommandAndReceiveResponse` in all channels